superdesk/default_settings.py

```diff
@@ -571,5 +572,12 @@ def celery_queue(name):
 XMPP_AUTH_DOMAIN = env('XMPP_AUTH_DOMAIN', 'Superdesk')
 
 
+#: webhooks (push system for feeding services) 
+#: dict were key is provider name and value is webhook endpoint
+#: "webhook_" will always be prefixed to endpoint name to avoid name conflicts
+#: e.g. {'wufoo NTB': 'wufoo'} will create webhook_wufoo endpoint
+FEEDING_WEBHOOKS = {}
```

Comment:
imo we should introduce an api for this, like we have to register ingest providers, outputs etc.